tests/error_003.ts.out

```diff
@@ -0,0 +1,6 @@
+[96m[WILDCARD]deno/tests/error_003.ts[0m:[93m1[0m:[93m1[0m - [91merror[0m[90m TS2552: [0mCannot find name 'consol'. Did you mean 'console'?
+
+[30;47m1[0m consol.log("hello world!");
+[30;47m [0m [91m~~~~~~[0m
+
+  [96m$asset$/globals.d.ts[WILDCARD]
```

Comment:
I'm not sure if I like all the term escapes here... but it's fine for now.

Reply:
At the moment, the matching will fail, because they are output with colours.  So I don't know if it is make the matching strip/ignore them or put them in the `.out`.